### PR TITLE
Feature/trap 38

### PR DESCRIPTION
**Pull request by vladimirnalitkin**

### CODE REVIEW

**Comment by vladimirnalitkin** [comment]
@bstarke

**Comment by vladimirnalitkin** [comment]
@ROHIT90 @msabirinsys

**Review by HungDo — APPROVED** [review]

File: src/test/java/com/insys/trapps/EngagementRepositoryTests.java [hunk 1]
```diff
@@ -0,0 +1,301 @@
...
+        );
+
+        Role testRole = Role.builder()
+                .name("Role 1")
+                .skills(testSkillSet)
+                .build();
+        roleRepository.save(testRole);
+
+        testEngagementList = Arrays.asList(
+                Engagement.builder()
```
Comment (HungDo) [inline src/test/java/com/insys/trapps/EngagementRepositoryTests.java]: there has to be a better way to do this

**Comment by vladimirnalitkin** [comment]
@ROHIT90 @msabirinsys

**Review by ROHIT90 — APPROVED** [review]

**Pull request merged**